spyder/plugins/help/confpage.py

```diff
@@ -43,6 +43,7 @@ def setup_page(self):
         features_group = QGroupBox(_("Additional features"))
         math_box = self.create_checkbox(_("Render mathematical equations"),
                                         'math')
+        # ??? Do we want to increase minimum sphinx requirement for Spyder?
```

Comment:
Yeah, this is a good idea. Perhaps increasing it to Sphinx 4 or 5 would be better, but I don't know enough about it to take an informed decision.

Reply:
I don't know enough either. It may be best to leave the lower limit where it currently is, at least for now.